setup-homebrew/post.mjs

```diff
@@ -0,0 +1,11 @@
+import {exec} from "@actions/exec"
+import core from "@actions/core"
+
+try {
+  await exec("bash", [
+    new URL("./post.sh", import.meta.url).pathname,
+    core.getInput("debug")
+  ])
+} catch (error) {
+  core.setFailed(error.message)
+}
```

Comment:
What's this needed for?

Reply:
Unfortunantely, GitHub does not support `post` actions in shell so a JS wrapped is needed.

Reply:
Why do we need post actions now?

Reply:
See post.sh review below.